robocop/checkers/comments.py

```diff
@@ -16,24 +10,21 @@
 
 
 class CommentChecker(VisitorChecker):
-    """ Checker for comments content. It detects invalid comments or leftovers like `todo` or `fixme` in the code. """
+    """Checker for comments content. It detects invalid comments or leftovers like `todo` or `fixme` in the code."""
+
     rules = {
-        "0701": (
-            "todo-in-comment",
-            "Found %s in comment",
-            RuleSeverity.WARNING
-        ),
+        "0701": ("todo-in-comment", "Found %s in comment", RuleSeverity.WARNING),
```

Comment:
This will make rules definitions a bit inconsistent but I think we can accept it since it will be replaced by class in the future anyway.